sdks/python/apache_beam/examples/wordcount.py

```diff
@@ -46,15 +46,15 @@ def word_extractor():
     The processed element.
   """
   line_splitter = re.compile(r'[A-Za-z\']+')
-  context, args, kwargs = yield
-  while True:
+  for context, _, _ in inputs:
     text_line = context.element.strip()
     if not text_line:
       context.aggregate_to(empty_line_aggregator, 1)
     words = line_splitter.findall(text_line)
     for w in words:
       context.aggregate_to(average_word_size_aggregator, len(w))
-    context, args, kwargs = yield words
+    yield words
```

Comment:
I'm fine with that, as long as it's in the contract to always return an iterator.